c2corg_ui/static/js/alertservice.js

```diff
@@ -120,11 +120,13 @@ app.Alerts.prototype.formatErrorMsg_ = function(response) {
   if (len > 1) {
     var msg = '<ul>';
     for (var i = 0; i < len; i++) {
-      msg += '<li>' + this.filterStr_(errors[i]['description']) + '</li>';
+      if (errors[i]['description'] && errors[i]['name']) {
```

Comment:
Do you really need to check that ``description`` and ``name`` are set? Is it possible they are not?